modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,93 @@ If you have installed the {gitops-title} Operator in the default namespace, run
 $ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
 ----
 
+[id="new-features-1-15-0_{context}"]
+== New features
+
+* With this update, {gitops-title} supports the addition of extra volumes and volume mounts to the `ApplicationSet` Controller components. You can configure these additional resources by specifying the `spec.applicationSet.volumes` and `spec.applicationSet.volumeMounts` fields in the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-5739[GITOPS-5739]
++
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-example
+spec:
+  applicationSet:
+    volumes:
+      - name: <volume-name> #<1>
+        emptyDir: {} #<2>
+    volumeMounts:
+      - name: <volume-name> #<3>
+        mountPath: <mount-path> #<4>
+----
+<1> Specifices the name of the `volume` within the Argo CD `ApplicationSet` Controller.
+<2> Specifies the volume type as an `emptyDir`, which creates an empty directory in the pod.
+<3> Specifices the name of the `volumeMounts` within the Argo CD `ApplicationSet` Controller.
```

Comment:
```suggestion
<1> Specifies the name of the `volume` for the Argo CD `ApplicationSet` Controller.
<2> Specifies the volume type. It can be any Kubernetes supported volume type, such as emptyDir, hostPath, configMap, etc.
<3> Specifies the name of the volume to be mounted, which must match the name of a volume defined in volumes.
```

Reply:
Fixed.